sections/appendix/blocking_time_formula/main.tex

```diff
@@ -0,0 +1,227 @@
+    b(2, T + 1) \quad &= \quad c(2, T + 1) + p_s b(2, T) + p_a b(2, T + 2) \\
+    & \ \, \vdots \nonumber \\
+    b(M - 1, N) \quad &= \quad c(M, N - 1) + b(M, N-1) \\ 
+    b(M, T) \quad &= \quad c(T, N) + p_s b(T-1, N) + p_a b(T, N+1) \\
+    & \ \, \vdots \nonumber \\
+    b(M, N) \quad &= \quad c(M, N) + b(M, N-1) \label{eq:last_eq_of_blocking_general}
+\end{align}
+
+The equivalent matrix notation of the linear system of equations 
+(\ref{eq:first_eq_of_blocking_general}) - (\ref{eq:last_eq_of_blocking_general})
+is given by \(Zx=y\), where:
+\begin{equation}
+    \scalebox{0.73}{\(
+        Z = 
+        \begin{pmatrix}
+            -1 & p_a & 0 & \dots & 0 & 0 & 0 & 0 & 0 & \dots & 0 & 0 \\ %(1,T)
+            p_s & -1 & p_a & \dots & 0 & 0 & 0 & 0 & 0 & \dots & 0 & 0 \\ %(1,T+1)
+            0 & p_s & -1 & \dots & 0 & 0 & 0 & 0 & 0 & \dots & 0 & 0 \\ %(1,T+2)
+            \vdots & \vdots & \vdots & \ddots & \vdots & \vdots & \vdots & 
+            \vdots & \vdots & \ddots & \vdots & \vdots \\ 
+            0 & 0 & 0 & \dots & 1 & -1 & 0 & 0 & 0 & \dots & 0 & 0 \\ %(1,N)
+            p_s & 0 & 0 & \dots & 0 & 0 & -1 & p_a & 0 & \dots & 0 & 0 \\ %(2,T)
+            0 & 0 & 0 & \dots & 0 & 0 & p_s & -1 & p_a & \dots & 0 & 0 \\ %(2,T+1)
+            \vdots & \vdots & \vdots & \ddots & \vdots & \vdots & \vdots & 
+            \vdots & \vdots & \ddots & \vdots & \vdots \\ 
+            0 & 0 & 0 & \dots & 0 & 0 & 0 & 0 & 0 & \dots & 1 & -1 \\ %(M,N)
+        \end{pmatrix},
+        x = 
+        \begin{pmatrix}
+            b(1,T) \\
+            b(1,T+1) \\
+            b(1,T+2) \\
+            \vdots \\
+            b(1,N) \\
+            b(2,T) \\
+            b(2,T+1) \\
+            \vdots \\
+            b(M,N) \\
+        \end{pmatrix}, 
+        y= 
+        \begin{pmatrix}
+            -c(1,T) \\
+            -c(1,T+1) \\
+            -c(1,T+2) \\
+            \vdots \\
+            -c(1,N) \\
+            -c(2,T) \\
+            -c(2,T+1) \\
+            \vdots \\
+            -c(M,N) \\
+        \end{pmatrix}
+    \)} \tag{\ref{eq:general_algebaric_approach_blocking_time} revisited}
+    \end{equation}
+
+    The elements of the matrix \(Z\) can be acquired using \(Z_{ij}\) defined in 
+    equation (\ref{eq:general_mapping_function_of_blocking_matrix}) where \(i\) 
+    and \(j\) are states \((u_i, v_i), (u_j, v_j) \in S_b\) 
+    (\ref{eq:set_of_blocking_states}).
+
+    \begin{equation}
+    Z_{ij} = 
+    \begin{cases}
+        p_a, & \textbf{if } j = i + 1 \textbf{ and } v_i \neq N \\
+        \\
+        p_s, & \textbf{if } j = i - 1 \textbf{ and } v_i \neq N, v_i \neq T \\
+            & \textbf{or } j = i - N + T \textbf{ and } u_i \geq 2,\,v_i = T \\
+        \\
+        1, & \textbf{if } j = i - 1 \textbf{ and } v_i = N \\
+        \\
+        -1, & \textbf{if } i = j \\
+        \\
+        0, & \textbf{otherwise} \\
+    \end{cases}
+    \tag{\ref{eq:general_mapping_function_of_blocking_matrix} revisited}
```

Comment:
```suggestion
    \begin{cases}
        p_a, & \textbf{if } j = i + 1 \textbf{ and } v_i \neq N \\
        p_s, & \textbf{if } j = i - 1 \textbf{ and } v_i \neq N, v_i \neq T \\
            & \textbf{or } j = i - N + T \textbf{ and } u_i \geq 2,\,v_i = T \\
        1, & \textbf{if } j = i - 1 \textbf{ and } v_i = N \\
        -1, & \textbf{if } i = j \\
        0, & \textbf{otherwise} \\
    \end{cases}
    \tag{\ref{eq:general_mapping_function_of_blocking_matrix} revisited}
```